logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -65,7 +65,12 @@ data class Conversation(
 
     sealed class ProtocolInfo {
         object Proteus : ProtocolInfo()
-        data class MLS(val groupId: String, val groupState: GroupState, val epoch: ULong) : ProtocolInfo() {
+        data class MLS(
+            val groupId: String,
+            val groupState: GroupState,
+            val epoch: ULong,
+            val keyingMaterialLastUpdate: ULong
```

Comment:
This can be an `Instant` at this level? Then we don't need think about what the ULong represents: seconds milliseconds...